deployment-configuration/codefresh-template-dev.yaml

```diff
@@ -1,4 +1,12 @@
 steps:
+  prepare_deployment:
```

Comment:
this is not needed anymore, and should use python 3.9

Reply:
yes, something went wrong with the merge of develop. fixing it